### ISSUE TITLE
Nagstamon -> icinga ack service sends wrong paramater formatting

### ISSUE THREAD

**Issue by jpptinsley** [HenriWahl/Nagstamon#312]
Nagstamon sends "send_notification=on" when it should send a notification, Icinga says: 
"cmd.cgi requires 0|1 vars, if not set, default is 1. View Bug #2926[0] for additional information." [1]

[0] https://dev.icinga.org/issues/2926
[1]http://docs.icinga.org/latest/en/cgiparams.html

Nagastamon sends:
`cmd_typ=34&cmd_mod=2&host=hostname&service=Local+Backup&com_author=jtinsley&com_data=acknowledged&btnSubmit=Commit&send_notification=on`

Icinga web interface sends:
` secmd_typ=34&cmd_mod=2&hostservice=hostname%5ELocal+Backup&com_author=jtinsley&com_data=TEST&end_time=2016-09-13+20%3A09%3A15&send_notification=1&btnSubmit=Commitnds:`

**Comment by jpptinsley** [comment]
My boss just rebuilt, after editing Servers/Icinga.py, but it's still not working as we expect -- it doesn't appear to be sending the notification;

`cmd_typ=34&cmd_mod=2&host=hostname-appa&service=check_debian_updates&com_author=jtinsley&com_data=TEST&btnSubmit=Commit&send_notification=1`

**Comment by HenriWahl** [comment]
Applied you change but can't test due to no Icinga 1 server available - can you do?

**Comment by jpptinsley** [comment]
@HenriWahl Thanks. Having trouble getting it to build, can you confirm dependencies? I'm building on a system running Ubuntu Xenial (mate).

**Comment by HenriWahl** [comment]
What error messages do you get? Did you install the typical Debian packaging packages?

**Comment by jpptinsley** [comment]
Yup, got both of those installed. Running the 'build.py' from the build directory is prompting:
```
** (python3.5:19276): WARNING **: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
Qt: Session management error: None of the authentication protocols specified are supported
Nagstamon.QUI (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: Nagstamon.QUI (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: Nagstamon.QUI
Traceback (most recent call last):
  File "/usr/lib/python3.5/unittest/loader.py", line 462, in _find_test_path
    package = self._get_module_from_name(name)
  File "/usr/lib/python3.5/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/home/jtinsley/development/Nagstamon/.pybuild/pythonX.Y_3.5/build/Nagstamon/QUI/__init__.py", line 6255, in <module>
    dbus_connection = DBus()
  File "/home/jtinsley/development/Nagstamon/.pybuild/pythonX.Y_3.5/build/Nagstamon/QUI/__init__.py", line 6087, in __init__
    dbus_sessionbus = SessionBus(dbus_mainloop)
  File "/usr/lib/python3/dist-packages/dbus/_dbus.py", line 211, in __new__
    mainloop=mainloop)
  File "/usr/lib/python3/dist-packages/dbus/_dbus.py", line 100, in __new__
    bus = BusConnection.__new__(subclass, bus_type, mainloop=mainloop)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 122, in __new__
    bus = cls._new_for_bus(address_or_type, mainloop=mainloop)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
E: pybuild pybuild:274: test: plugin distutils failed with: exit code=1: cd /home/jtinsley/development/Nagstamon/.pybuild/pythonX.Y_3.5/build; python3.5 -m unittest discover -v 
dh_auto_test: pybuild --test -i python{version} -p 3.5 --dir . returned exit code 13
debian/rules:7: recipe for target 'binary' failed
make: *** [binary] Error 25
```

**Comment by HenriWahl** [comment]
Never seen this message... I always built on Debian 8 - quite possible that a newer Mint behaves differently. At the moment there is nothing I can do there but I am about to release some binaries soon.

**Comment by HenriWahl** [comment]
Binaries are now available for 2.0.1 testing

**Comment by jpptinsley** [comment]
@HenriWahl Sorry for the delay. We've been using the beta in anger for a while now, and it's working brilliantly, thanks. No issues. Closing now
